test/orchard/apropos_test.clj

```diff
@@ -3,13 +3,19 @@
    [clojure.repl :as repl]
    [clojure.string :as str]
    [clojure.test :refer [are deftest is testing]]
-   [orchard.apropos :refer [find-symbols]]
+   [orchard.apropos :as sut :refer [find-symbols]]
    [orchard.meta :refer [var-name var-doc]]))
 
 (def ^{:doc "Test1. Test2. Test3."} public-var [1 2 3])
 
 (defn some-random-function [])
 
+(deftest apropos-sort-test
```

Comment:
Added some extra tests, they passed before the fix anyway.

https://github.com/clojure-emacs/orchard/issues/128 seems hard to repro since apropos results depend on ones's classpath.